### PR TITLE
Catalog page

### PR DESCRIPTION
**Pull request by npeltier**
* Catalog page layout (with catalog block)
* Sidenav block

Resolves: 
 - MWPW-136872
 - MWPW-136866
 - MWPW-132393
 - MWPW-137257
 - MWPW-134233
 - MWPW-139915
 - MWPW-141104

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.live/products/special-offers
- After: https://commercev3--cc--adobecom.hlx.live/products/special-offers
- Before: https://main--cc--adobecom.hlx.page/drafts/ilyas/catalog
- After: https://commercev3--cc--adobecom.hlx.page/drafts/ilyas/catalog

### CODE REVIEW

File: creativecloud/deps/merch-spectrum.min.js
Comment (yesil) [inline creativecloud/deps/merch-spectrum.min.js:1]: this file is not needed anymore, we can delete it.

File: creativecloud/deps/lit-all.min.js
Comment (yesil) [inline creativecloud/deps/lit-all.min.js:1]: we can delete this file as well

File: creativecloud/blocks/sidenav/sidenav.js [hunk 1]
```diff
@@ -61,42 +61,38 @@ const getTypes = (arrayTypes) => {
 };
 
 const appendFilters = async (root, link, explicitCategoriesElt) => {
-  try {
```
Comment (npeltier) [inline creativecloud/blocks/sidenav/sidenav.js]: @yesil are we good with this? try catch { //} sounds stupid to me

File: creativecloud/blocks/sidenav/sidenav.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+      });
+      checkbox.append(item.name);
+      tag.append(checkbox);
+    }
+  });
+  return tag;
+};
+
+const appendFilters = async (root, link, explicitCategoriesElt) => {
+  const resp = await fetch(link);
```
Comment (yesil) [inline creativecloud/blocks/sidenav/sidenav.js:64]: if the fetch fails, the exception must be caught.

File: creativecloud/blocks/sidenav/sidenav.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  const title = el.querySelector('h2')?.textContent.trim();
+  const rootNav = createTag('merch-sidenav', { title });
+  const searchText = el.querySelector('p > strong')?.textContent.trim();
+  appendSearch(rootNav, searchText);
+  // eslint-disable-next-line prefer-const
+  let [endpoint, resourcesLink] = el.querySelectorAll('a');
+  if (endpoint) {
+    endpoint = localizeLink(endpoint.textContent.trim(), null, true);
+    const explicitCategories = el.querySelector('ul');
+    await appendFilters(rootNav, endpoint, explicitCategories);
```
Comment (yesil) [inline creativecloud/blocks/sidenav/sidenav.js:144]: maybe catch the exception here ?

**Review by yesil — APPROVED** [review]

**Review by 3ch023 — APPROVED** [review]

**Pull request merged**